src/Footer.js

```diff
@@ -1,6 +1,5 @@
 import React, { Component } from "react";
-import './Footer-with-map.css';
-import Map from '../googleMap/GoogleMap';
+import Map from './googleMap/GoogleMap';
```

Comment:
Did you move GoogleMap to other folder? Or it's mistake? (I mean relative path to GoogleMap)